codec/src/main/java/io/netty/handler/codec/DefaultHeaders.java

```diff
@@ -79,19 +119,47 @@ public T get(T name, T defaultValue) {
     }
 
     @Override
-    @SuppressWarnings("unchecked")
     public T getAndRemove(T name) {
```

Comment:
I don't remember why to be honest. Looking from the latest changeset, I can't find the double `for (;;)`, so let me just assume this has been sorted out properly. :-)